README.md

```diff
@@ -13,3 +16,54 @@ The enclosed .yaml file leverages the OpenApi Specification, enabling consumers
 The Origin platform publishes all trade data according to Airbrush, more information can be found on the Orign API documentation page at: https://login2.originmarkets.com/api/trades/
 
 Please contact airbrush@originmarkets.com if you have any questions or feedback.
+
+## Trying out the Origin Termsheet
+
+### Installation
+
+Start by installing [python](https://www.python.org/downloads/) onto your computer as the command line
+tool to try out Airbrush is written in python. To make sure that this has worked, open a terminal program
+and run:
+
+```sh
+python -V
+```
+
+You should see an output of `Python 3.x.x` with each `x` being a specific number depending on the version of
+python that you downloaded. Once python is installed, clone the repo using [git](https://git-scm.com/) and
+change your working directory to the project:
+
+```sh
+git clone https://github.com/OriginMarkets/Airbrush.git
+cd Airbrush
+```
+
+Now that the repo is downloaded, get the project dependencies with the following
+[pip](https://www.python.org/downloads/) command:
+
+```sh
+pip install -r requirements.txt
```

Comment:
the convention would be to not make the user install these dependencies system-wide, rather, to install them inside a virtual environment. instructions for that look like:

Create and start a virtual env
```sh
python3 -m venv env1
source env1/bin/activate
```

Install dependencies
```sh
pip install --upgrade pip
pip install -r requirements.txt
```

Reply:
Yup makes sense. Was trying to keep the instructions simple but its not what I would do either.